app/views/design/things-to-consider-in-defence/index.njk

```diff
@@ -74,21 +76,26 @@
     <li>cause exceptional damage to UK or allied military capability</li>
   </ul>
 
+  <p>It will be clear that you are working on information or systems that are TOP SECRET.</p>
+
+
   <h3>How to label information in Defence</h3>
 
-  <p>You need to label information correctly so that people understand how to protect and share it.</p>
+  <p>You need to label information correctly so that people understand how to protect and share it.
```

Comment:
@miaodon we need to ensure HTML tags are closed correctly, otherwise it invalidates the page and can cause rendering issues. Easy enough to miss sometimes. I will add a HTML validation NPM package to check for these issues.